init_processes.sh

```diff
@@ -5,11 +5,10 @@ if [ ! -f "settings/settings.json" ]; then
     exit 1;
 fi
 
-if [ -z "$UUID" ]; then
-    echo "UUID not found in .env - autopopulating to 'generated-uuid'!";
-    export uuid="${UUID:-generated-uuid}"
+if [ ! -z "$UUID" ]; then
+    echo "UUID found in .env - using provided UUID!";
+    export uuid="${UUID}"
```

Comment:
This was a misplacement of `fi` on my end during debugging. However, now it won't set a default UUID if none is set in `.env`, right?

Reply:
I tested that, if it is none then the add-snapshotter command automatically generates a random uuid. Should be good.

Reply:
So, the idea was to intentionally generate `generated-uuid` so it could be used with default settings. This was useful for the CI/CD setup I am building. A random UUID isn't useful as it could generating one on every startup, right?